src/post/PostSingleContent.js

```diff
@@ -36,38 +50,68 @@ const PostSingleContent = ({ content, toggleBookmark, bookmarks }) => {
                 name={bookmarks[content.id] ? 'bookmark' : 'bookmark_border'}
               />
             </a>
-
           </span>
         </div>
 
-        <div className="PostSingleContent__content mt-4">
+        <div className="PostSingleContent__content mb-3">
           <h1 className="mvl">{content.title}</h1>
           <Body body={content.body} jsonMetadata={content.json_metadata} />
-          { jsonMetadata.tags &&
-            <div className="my-2">
-              { jsonMetadata.tags.map(tag => (
-                <span key={tag}>
-                  <Link to={`/hot/${tag}`} className="tag tag-default">{tag}</Link>{ ' ' }
-                </span>
-              ))}
-            </div>
-          }
-          <AuthorBio authorName={content.author} />
         </div>
+
+        {jsonMetadata.tags &&
+          <div className="mb-3">
+            {jsonMetadata.tags.map(tag => (
+              <span key={tag}>
+                <Link to={`/hot/${tag}`} className="btn btn-sm btn-secondary">
+                  {tag}
+                </Link>
+                { ' ' }
+              </span>
+            ))}
+          </div>
+        }
       </div>
 
-      { content.children > 0 && (
-        <div className="PostSingleContent__replies py-3">
-          <div className="container">
-            <h1><Icon name="reply" lg /> Comments ({content.children})</h1>
+      <MenuPost
```

Comment:
It's because MenuPost is now not fixed on screen but appear between the end of article and Author bio